src/js/components/App.js

```diff
@@ -1,39 +1,95 @@
 import MenuInput from './MenuInput.js';
 import MenuList from './MenuList.js';
 import Count from './Count.js';
+import { $ } from '../utils/dom.js';
+import { UPDATE_MSG, DELETE_MSG } from '../constants/index.js';
+import { store } from '../store/index.js';
 
 function App() {
-  // NOTE:
-  // enuInfo라는 이름이 적당한지는 모르겠음
-  // 내부에 menu title과 id 값을 가진 object들을 요소로 가짐
-  let menuInfo = [];
   let menuId = 0;
+  let currentCategory = 'espresso';
+  let menu = {
+    espresso: [],
+    frappuccino: [],
+    blended: [],
+    teavana: [],
+    desert: [],
+  };
+
+  const count = new Count();
+  const menuList = new MenuList();
 
-  const setState = updatedMenuList => {
-    menuInfo = updatedMenuList;
+  this.init = () => {
+    if (localStorage.getItem('menu')) menu = store.getItem();
```

Comment:
```suggestion
   menu = store.getItem(menu);
```
`store.getItem`이 `defaultValue`를 파라미터로 받을 수 있다면 이렇게 선언 할 수 있을 것 같아요